### PR TITLE
a basic refactoring (#15)

### PR DESCRIPTION
**Pull request by mkjasinski**

### CODE REVIEW

File: tests/test_ohmycomposer.py [hunk 1]
```diff
@@ -70,9 +80,43 @@ def test_should_return_correct_env_passed_by_env_variable(self):
...
+        os.environ['CONFIGURATION_VARIANT'] = 'test'
+        ohmycomposer = OhMyComposer()
+
+        # when
+        config_variant = ohmycomposer.configuration_variant()
+
+        # then
+        assert config_variant == "test"
+
+        # cleanup
```
Comment (devorin) [inline tests/test_ohmycomposer.py:99]: extract to method? or some cleanup after test case

File: tests/test_ohmycomposer.py [hunk 2]
```diff
@@ -70,9 +80,43 @@ def test_should_return_correct_env_passed_by_env_variable(self):
...
+        ohmycomposer = OhMyComposer(bucket='bucket', project_id='project', project_prefix='prefix', env=Env.DEV)
+
+        # when
+        config_variant = ohmycomposer.configuration_variant()
+
+        # then
+        assert config_variant == 'default'
+
     def test_should_get_correct_values_passed_by_parameters(self):
         # setup
```
Comment (devorin) [inline tests/test_ohmycomposer.py:118]: why not "standard" #given?

File: tests/test_ohmycomposer.py (hunk 1 above)
Comment (mkjasinski) [reply tests/test_ohmycomposer.py:99]: yes, clean_up method we'll be good solution

File: tests/test_ohmycomposer.py (hunk 2 above)
Comment (mkjasinski) [reply tests/test_ohmycomposer.py:118]: yes, my bad, of course, #given

**Review by devorin — APPROVED** [review]

**Review by Dominos — APPROVED** [review]

**Pull request merged**